cosima_cookbook/database.py

```diff
@@ -371,17 +362,10 @@ def index_experiment(experiment_dir, session=None, client=None, update=False, pr
                                                   v.name, v.long_name,
                                                   v.standard_name, v.units)
 
-    if prune:
-        for f in missing_files:
-            if delete:
-                session.delete(missing_files[f])
-            else:
-                missing_files[f].present = False
-
     session.add_all(results)
     return len(results)
 
-def build_index(directories, session, client=None, update=False, prune=False, delete=False):
+def build_index(directories, session, client=None, update=False, prune=True, delete=True):
     """Index all netcdf files contained within experiment directories.
```

Comment:
We probably should explain `update`, `prune` and `delete` in this docstring...

Reply:
Done